main/autogen/autogen.cc

```diff
@@ -674,4 +676,331 @@ void ParsedFile::classlist(core::Context ctx, vector<string> &out) {
     }
 }
 
+NamedDefinition ParsedFile::toNamed(core::Context ctx, DefinitionRef def) {
```

Comment:
why is this a member of `ParsedFile` rather than a member of `DefinitionRef` or, even better a static method?

Reply:
Made this `static NamedDefinition::fromDef`